raft_test.go

```diff
@@ -1490,13 +1490,17 @@ func TestRaft_LeaderLeaseExpire(t *testing.T) {
 	c.Disconnect(follower.localAddr)
 
 	// Watch the leaderCh
-	select {
-	case v := <-leader.LeaderCh():
-		if v {
-			c.FailNowf("should step down as leader")
+	timeout := time.After(conf.LeaderLeaseTimeout * 2)
```

Comment:
Would we be able to not use this timeout by implementing a retry testutil? Would love to see that improvement. :)

Reply:
Sadly, a retry testutil helper function wouldn't make this necessarily cleaner, since we are also blocking on a channel receive.